src/amp/components/BodyArticle.tsx

```diff
@@ -34,20 +35,19 @@ const bulletStyle = (pillar: CAPIPillar) => css`
 	}
 `;
 
-const body = (pillar: CAPIPillar, designType: DesignType) => {
-	const defaultStyles: DesignTypesObj = designTypeDefault(
-		palette.neutral[100],
-	);
-
-	// Extend defaultStyles with custom styles for some designTypes
-	const designTypeStyle: DesignTypesObj = {
-		...defaultStyles,
-		Comment: palette.opinion[800],
-		AdvertisementFeature: palette.neutral[86],
-	};
+const decideBackground = (design: Design, pillar: CAPIPillar): string => {
+	if (pillar === 'labs') return palette.neutral[86];
+	switch (design) {
+		case Design.Comment:
+			return palette.opinion[800];
+		default:
+			return palette.neutral[100];
+	}
+};
```

Comment:
`designTypeDefault` was throwing odd typescript errors so to fix these, and also because we want to also use `pillar` in this decision now too, I refactored this logic away from `designTypeDefault`

This will also make moving over to an Editorial Palette easier